hail/python/hail/methods/impex.py

```diff
@@ -656,7 +659,8 @@ def import_bed(path, reference_genome='default', skip_invalid_intervals=False) -
                                                    'f2': tint32, 'f3': tstr,
                                                    'f4': tstr},
                      comment=["""^browser.*""", """^track.*""",
-                              r"""^\w+=("[\w\d ]+"|\d+).*"""])
+                              r"""^\w+=("[\w\d ]+"|\d+).*"""],
```

Comment:
This looks good, but my one concern is if someone overrides one of these options that we know are correct for bed files.

Reply:
What do you mean? If someone passes `reference_genome='GRCh38'` it will correctly go to import_bed, but anything not in the signature will go on to `import_table` - if down the line, someone adds `reference_genome` to import_table, that will be a problem, but not sure if that is fixable here.

Reply:
If someone overrides these they'll get an error that you tried to pass the same argument twice. Maybe better to explicitly check for these options, but it seems hard to really screw yourself here

Reply:
Ohhhh you mean like `comment=`? 🤔

Reply:
yeah